core/pom.xml

```diff
@@ -321,20 +321,9 @@ THE SOFTWARE.
         <version>1.19</version>
     </dependency>
     <dependency>
-      <groupId>javax.mail</groupId>
-      <artifactId>mail</artifactId>
-      <version>1.4.4</version>
-      <exclusions>
-        <exclusion><!-- using a patched version -->
-          <groupId>javax.activation</groupId>
-          <artifactId>activation</artifactId>
-        </exclusion>
-      </exclusions>
-    </dependency>
-    <dependency>
-      <groupId>org.jvnet.hudson</groupId>
-      <artifactId>activation</artifactId>
```

Comment:
why is this removed?

Reply:
Good question. I'm not even sure what was patched originally, but it's a transitive dependency of jakarta.mail already.